jekyll/_ccie/quick-start.md

```diff
@@ -1,71 +1,81 @@
 ---
 layout: enterprise
 section: enterprise
-title: "User Quick Start"
+title: "Setting Up Projects"
 category: [documentation]
 order: 1
-description: "A light overview on how to get started with CircleCI Enterprise."
+description: "Overview of getting started with CircleCI Enterprise."
 ---
 
-This guide is for developers using existing installations of CircleCI Enterprise.
-Instructions for installing and setting up Enterprise can be found [here]({{site.baseurl}}/enterprise/overview/).
-For developers testing and deploying their code with CircleCI Enterprise, the
-experience extremely similar to using circleci.com, so much of the information in the
-[docs](https://circleci.com/docs/) and [support forum](https://discuss.circleci.com)
-for circleci.com are applicable. There are a few potentially
-confusing differences between CircleCI Enterprise and circleci.com though, so this guide
-provides a starting point specifically for users of Enterprise.
-
-Let's quickly cover a few of the first things you might want to try on CircleCI Enterprise. If you
-are curious about the differences between your CircleCI Enterprise installation and public circleci.com,
-you can skip to the [differences from circleci.com]({{site.baseurl}}/enterprise/differences/).
-
-<ul><li><b>Add your first project:</b> Setting projects up to build on CircleCI Enterprise is really easy.
-Just:
-<ol>
-    <li>Sign up for an account on your CircleCI Enterprise instance (make sure you have a GitHub Enterprise account first)</li>
-    <li>Select the GitHub user or organization of the project you want to build</li>
-    <li>Find your project in the list of projects and click "Build project" (or "Watch project" if it is already being built on CircleCI)</li>
-</ol>
-<p>That's it! If the inferred steps don't do quite what you want, try adding a `circle.yml` file as described below. You
-can add more projects at any time by clicking the "Add Projects" link on the left sidebar from the CircleCI Enterprise app.
-You can also check out a video walkthrough here:</p>
+This guide describes how to set up a project and provides a primer on configuration troubleshooting in CircleCI Enterprise for developers, as follows:
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+To add a project, you must have a GitHub or GitHub Enterprise account.
+
+## Adding a Project
+
+Complete the following steps to set up a project on CircleCI Enterprise.
+
+1. Click the Open link on the https://localhost:8800/dashboard page, or navigate to the IP address for the CircleCI application as installed by your administrator. The Welcome page appears with a Get Started button. ![CircleCI Enterprise Welcome]({{ site.baseurl }}/assets/img/docs/ccie_welcome.png)
```

Comment:
Change `localhost:8800` to some kind of reference to whatever their installed domain/IP will be (and it would not generally be at 8800 - that's the admin panel that users would not care about).